### ISSUE TITLE
Problem with times listed under the Runs tab

### ISSUE THREAD

**Issue by jforthofer** [firelab/windninja-mobile#145]
In the runs tab, the date/time listed under a run for "Submitted" and "Updated" are wrong.  There must be some problem in the calculation of that time.

**Comment by fspataro** [comment]
Looks like the server time is set to UTC --- which makes sense for a cloud vm since you really don't know where it is... We had an assumption about it's timezone but also didn't think it would generating UTC time by default so it was double converting.  I'm going to explicitly set the job message times to UTC then we can calculate the display time correctly

**Comment by fspataro** [comment]
Also the "Point Six (test)" job will initially have the current time for both values until the user presses the "status" button.  We're just storing the "id" in the app not the whole job file so it doesn't know the times until it makes the first request for the job details. It updates correctly.

**Comment by fspataro** [comment]
Seems to be displaying correct GMT times now.  I'm going to keep GMT as the display for this build...but everything is in place for us to display whatever time zone we want. There's a method on the time library to "guess" the user's current time zone.  It works fine in the current browsers but right now it's unlcear if that works for the phones and actually detects moving into a different time zone.   For an enhancement in the future we could create a UI to allow the user to pick what time zone to display and that would just be UI work at this point.

**Comment by fspataro** [comment]
Sorry, reopening so Natalie or Jason can close after verification--- added correct tag.

**Comment by fspataro** [comment]
Yeah, we were thinking a user setting for time values:  UTC, current user location, DEM, maybe a specific US timezone, etc.
